x-pack/plugins/ml/server/routes/results_service.ts

```diff
@@ -71,6 +76,19 @@ function getPartitionFieldsValues(legacyClient: ILegacyScopedClusterClient, payl
   return rs.getPartitionFieldsValues(jobId, searchTerm, criteriaFields, earliestMs, latestMs);
 }
 
+function getCategorizerStats(context: RequestHandlerContext, params: any, query: any) {
```

Comment:
this needs updating to use `legacyClient `

Reply:
Updated in https://github.com/elastic/kibana/pull/74592/commits/a94ecaf246b7f84168eb296b5c2c1a75471353cc